### ISSUE TITLE
[Ubuntu] Zip game uses libssl version that is too old

### ISSUE THREAD

**Issue by SunCatMC** [YoYoGames/GameMaker-Bugs#6365]
### Description
I want to use zip compilation because that makes the assets easily accessible and modifyable. But that encounters an issue `error while loading shared libraries: libcrypto.so.1.0.0: cannot open shared object file: No such file or directory`. To resolve that issue, you gotta find a way to install that object - but turns out you can't find that by searching Distribution:any at https://packages.ubuntu.com - the oldest you get is 1.1.1f, which doesn't work for this usecase. Instead you gotta find a random Stackexchange answer that tells you where you can install that package from https://superuser.com/questions/1800031/unable-to-instal-libcrypto-so-1-0-0-on-ubuntu-20-04-6
It would be better to at least upgrade it to 1.1.1, so that at least ubuntu 20.04LTS doesn't have any hoops tojump through

TLDR to make zip games working on ubuntu you have to:
- try to edit config via `sudo nano /etc/apt/sources.list`
- if there are sources listed in it, add line `deb http://security.ubuntu.com/ubuntu xenial-security main`
-- if istead it says `# Ubuntu sources have moved to /etc/apt/sources.list.d/ubuntu.sources`, then:
-- open that config via `sudo nano /etc/apt/sources.list.d/ubuntu.sources`
-- add the following lines:
```
Types: deb
URIs: http://security.ubuntu.com/ubuntu
Suites: xenial-security
Components: main
Signed-By: /usr/share/keyrings/ubuntu-archive-keyring.gpg
```
-  download package info via `sudo apt-get update`
- install the package via `sudo apt-get install libssl1.0.0`

on other distros installing libssl1.0.2 or openssl1.0.2 also works

sorry if this should be a feature request, i wanted to submit my project as an example of the issue

### Steps To Reproduce
- Grab project
- Compile project as zip
- Run project on most machines - see libcrypto error

### Which version of GameMaker are you reporting this issue for?
IDE v2024.600.0.579 Runtime v2024.600.0.605

### Which operating system(s) are you seeing the problem on?
Linux Ubuntu 24.04 LTS kernel 6.8.0.35

### Which platform(s) are you seeing the problem on?
Ubuntu

[bb0b1755-2a33-497b-b7de-ade8b1cb81dd](https://api.gamemaker.io/api/github/downloads/bb0b1755-2a33-497b-b7de-ade8b1cb81dd)
